### PR TITLE
Add fetch function & add better logging

### LINKED ISSUE #5: Implement a `fetch` command
The `fetch` command will download data from the Google Sheet and store each sheet as a TSV file with matching name in the `.cogs/` directory.

### PR DESCRIPTION
**Pull request by beckyjackson**
Resolves #5

Also, instead of printing everything, change messages to use `logging`.

## Logging

To print info- and warn-level logging messages (error- and critical-level messages are always printed), run any command with the `-v`/`--verbose` flag:

```
cogs [command & opts] -v
```

Otherwise, most commands succeed silently.

---

### `fetch`

Running `fetch` will sync the local `.cogs/` directory with all remote spreadsheet changes.

```
cogs fetch
```

This will download all sheets in the spreadsheet to that directory as `{sheet-title}.tsv` - this will overwrite the existing sheets in `.cogs/`, but will not overwrite the local versions specified by their path. As the sheets are downloaded, the fields are checked against existing fields in `.cogs/field.tsv` and any new fields are added with the default datatype of `cogs:text` (text string). Any sheets that have been added with `add` and then pushed to the remote sheet with `push` will be given their IDs in `.cogs/sheet.tsv`.

If a new sheet has been added to the Google spreadsheet, this sheet will be downloaded and added to `.cogs/sheet.tsv`. The default path for pulling changes will be the current working directory (the same directory as `.cogs/` is in). This path can be updated with `cogs mv`.

To sync the local version of sheets with the data in `.cogs/`, run `cogs pull`.

### CODE REVIEW

File: src/cogs/cli.py [hunk 1]
```diff
@@ -22,13 +23,15 @@ def version(args):
 
 def main():
     parser = ArgumentParser()
+    global_parser = ArgumentParser(add_help=False)
+    global_parser.add_argument("-v", "--verbose", help="Print logging", action="store_true")
```
Comment (jamesaoverton) [inline src/cogs/cli.py:28]: Can `-v` just be added to `parser` and be inherited by all its subparsers, without defining a new `global_parser`?

**Comment by jamesaoverton** [comment]
I haven't tested this, but the code looks good (I had one comment). Please merge when you're ready.

File: src/cogs/cli.py (hunk 1 above)
Comment (beckyjackson) [reply src/cogs/cli.py:28]: I tried that and it didn't work, so I searched for an answer and it looks like you have to create a separate "parent" - https://stackoverflow.com/questions/33645859/how-to-add-common-arguments-to-argparse-subcommands/33646419

**Comment by beckyjackson** [comment]
OK - I'm just updating `rm` to use the new logging and then I'll merge. Thanks!

**Pull request merged**